bundles/org.openhab.ui/web/src/js/routes.js

```diff
@@ -102,6 +102,7 @@ export default [
   },
   {
     path: '/setup-wizard/',
+    beforeEnter: [enforceAdminForRoute],
```

Comment:
I have tested this on a fresh, clean openHAB installation, and you have to create an admin account before entering the wizard so this does not cause problems.